### PR TITLE
Updated kubecon banners

### PR DESCRIPTION
**Pull request by kbarnard10**
Updating homepage button and community page banner with KubeCon Barcelona

### CODE REVIEW

File: layouts/community/list.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
 {{ end }}
 
 {{ define "post-hero" }}
-<a href="https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2018/"><img src="/images/kubecon_china_ad_2.jpg" alt="KubeConShanghai" width="49.9%" align="center"></a>
-<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/"><img src="/images/kubecon_na_ad_2.jpg" alt="KubeConSeattle" width="49.9%" align="right"></a>
+<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/"><img src="/images/kubecon_na_ad_2.jpg" alt="KubeConSeattle" width="49.9%" align="center"></a>
+<a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019/"><img src="/images/kubecon_na_ad_barcelona.jpg" alt="KubeConBarcelona" width="49.9%" align="right"></a>
```
Comment (jimangel) [inline layouts/community/list.html]: I can't find `kubecon_na_ad_barcelona.jpg` in https://github.com/kubernetes/website/tree/master/static/images

**Comment by xiangpengzhao** [comment]
/lgtm
as per my preview.

**Comment by xiangpengzhao** [comment]
One concern:
The date on the banner of KubeCon Barcelona is May 20-23 (including the first day for Community Co-Located Events). But the date on the banner of KubeCon Seattle is Dec 11-13 (**not including the first day for Community Co-Located Events**). Should we modify it?

File: content/en/_index.html [hunk 2]
```diff
@@ -44,12 +44,12 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/" button id="desktopKCButton">Attend KubeCon in Shanghai on Nov. 13-15, 2018</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/" button id="desktopKCButton">Attend KubeCon in Seattle on Dec. 11-13, 2018</a>
```
Comment (dankohn) [inline content/en/_index.html]: Dec. 10-13

**Comment by dankohn** [comment]
@xiangpengzhao You're correct. We like to use the date Dec 10-13 to include the pre-events and lightning talks the night before the main event.

**Comment by xiangpengzhao** [comment]
/lgtm cancel
/hold
@kbarnard10 please address the above comments on the date of KubeCon Seattle :)

**Comment by kbarnard10** [comment]
@xiangpengzhao We will be swapping out the Seattle banner next week so this can be merged in the meantime.

**Pull request closed without merging**